product-service/src/database/init.sql

```diff
@@ -0,0 +1,44 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS products
+(
+    id          uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    title       text NOT NULL,
+    description text,
+    price       integer
```

Comment:
for this use better numeric or any floating-point types, since can be the price for example 2.35$

Reply:
Thank you for the review. I agree with you!